docs/installation/README.md

```diff
@@ -2,19 +2,23 @@
 
 Installation can be daunting, so here are some notes.
 
+## Installing via docker
+
+If you have docker installed, you can bring up the Lute stack with a single command.  See [installing via docker](./install_docker.md).
+
+Docker requires very little setup, and is a great way to get started.
+
 ## Installing your own Apache/PHP/MySQL stack
 
 If you're more technical, and are setting up your own Apache/Php/MySQL stack instead of using a pre-packaged thing like MAMP/XAMPP/Whatever, I'll assume you can get that rolling, but here are some quick notes for [installing on plain Apache](./install_apache.md).
-
```

Comment:
No need to remove stuff that the PR doesn't need to touch.  It's fine, just fyi for other projects. :-)